eng/tools/sdk-suppressions/tsconfig.json

```diff
@@ -3,5 +3,9 @@
   "compilerOptions": {
     "outDir": "./dist",
     "allowJs": true
-  }
+  },
+  "include": ["src"],
+  "references": [
+    { "path": "../suppressions" }
```

Comment:
Why are you referencing the "suppressions" project?  I don't see it used anywhere in your TS code.

Reply:
Removed